operations/admin.go

```diff
@@ -171,3 +173,42 @@ func setServiceFlagValues(args []string, target bool, flags *model.APIServiceFla
 
 	return catcher.Resolve()
 }
+
+func viewSettings() cli.Command {
+	return cli.Command{
+		Name:   "get-settings",
+		Usage:  "view the evergreen configuration settings",
+		Action: doViewSettings(),
+	}
+}
+
+func doViewSettings() cli.ActionFunc {
+	return func(c *cli.Context) error {
+		confPath := c.Parent().String(confFlagName)
+
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		conf, err := NewClientSetttings(confPath)
+		if err != nil {
+			return errors.Wrap(err, "problem loading configuration")
+		}
+		client := conf.GetRestCommunicator(ctx)
+		defer client.Close()
+
+		settings, err := client.GetSettings(ctx)
+		if err != nil {
+			return errors.Wrap(err, "problem getting evergreen settings")
+		}
+
+		// marshal the struct into yaml so that the output comes out nicely indented
+		settingsYaml, err := yaml.Marshal(settings)
+		if err != nil {
+			return errors.Wrap(err, "problem marshalling evergreen settings")
+		}
+
+		grip.Info(settingsYaml)
```

Comment:
we should consider using the "plain formater" to avoid logging prefixing. 

You could also implement message.Composer for the settings object to avoid having to marshal this here? Not sure.

Reply:
Switched this to the plain sender